Mathlib/Data/List/Infix.lean

```diff
@@ -35,36 +33,23 @@ variable {l l₁ l₂ l₃ : List α} {a b : α} {m n : ℕ}
 
 /-! ### prefix, suffix, infix -/
 
-
 section Fix
 
+@[deprecated prefix_refl (since := "2024-08-15")]
 theorem prefix_rfl : l <+: l :=
-  prefix_refl _
+  prefix_refl
 
+@[deprecated suffix_refl (since := "2024-08-15")]
 theorem suffix_rfl : l <:+ l :=
-  suffix_refl _
+  suffix_refl
 
+@[deprecated infix_refl (since := "2024-08-15")]
 theorem infix_rfl : l <:+: l :=
```

Comment:
Wait, is upstream using `_refl` for the implicit argument version? That should be fixed

Reply:
Ah, indeed. I've removed the deprecations and will fix upstream.

Reply:
https://github.com/leanprover/lean4/pull/5077